R/parser.R

```diff
@@ -35,7 +35,7 @@ read_services <- function(file){
 
     w <- "RequestType"
 
-    while(tokens[i] != '}'){
+    while(tokens[i] != '}' && tokens[i] != ';'){
```

Comment:
We / I should add some more tests around parsing.

Reply:
Sounds good.